packages/roc-package-web-app-react/src/config/roc.config.js

```diff
@@ -26,6 +26,7 @@ export default {
                 style: [],
             },
             ssr: true,
+            redirectStatusCode: 301,
```

Comment:
Let's have the default as 302 to not introduce a breaking change here and we can go with `301` in the next breaking release.